web/docs/operators/show.md

```diff
@@ -17,7 +17,7 @@ show [<aspect>]
 ## Description
 
 The `show` operator offers introspection capabilities to look at various
-*aspects* of a Tenzir node.
+_aspects_ of a Tenzir node.
```

Comment:
Since when do we use underscores for emphasis?

Reply:
We don't, I think this was a byproduct of some config changes I made following the neovim 0.10 update.